docs/client_advanced.rst

```diff
@@ -451,6 +451,23 @@ In order to specify the nameservers to when resolving the hostnames,
     conn = aiohttp.TCPConnector(resolver=resolver)
 
 
+Changing source interface
+^^^^^^^^^^^^^^^^^^^
+
+By default, if you have multiple internet connections, aiohttp will choose the
```

Comment:
I don't think it's correct to imply that it's `aiohttp` that chooses the route, it's not true — the OS kernel chooses it. Moreover, even if you bind to an interface, I'm pretty sure this can still be intercepted via something like iptables and rerouted outside the application.
So let's try to rephrase this to be accurate in what actually happens.

Reply:
Ah, that's not entirely correct. https://stackoverflow.com/a/68906789/595220 says that this would guarantee that the traffic would go out of the said interface. But still, it's not routing since routing is an L3 concept and interfaces are lower on the OSI stack.

Reply:
I will rephrase that.